### PR TITLE
fixed responsiveness of footer.

### LINKED ISSUE #116: [Site] Footer re-work and Mobile responsiveness
**Current State of the GetNighthawk.dev site**
Currently, The footer has a few components (copyright text and social media logos) that are wrongly aligned in the desktop view as well as the mobile view.
chrome_fuT4082djy
chrome_nb33r6vaV6

**Desired State of the GetNighthawk.dev site**
Refer to the Figma link - https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs?node-id=6189%3A0

---

**Contributor Resources**

- See the [Contributing to getnighthawk.dev Website](https://github.com/layer5io/getnighthawk/blob/master/CONTRIBUTING.md)
- See the Project Site section of [GetNighthawk project doc](https://docs.google.com/document/d/1lHfMo4iIx2WXFZIspfHyxTsPR1T63_2IV5NUkgxoo0w)

### PR DESCRIPTION
**Pull request by Anita-ihuman**
Signed-off-by: Anita-ihuman <charlesanita403@gmail.com>

**Description**
Fixed the footer to be responsive on mobile and desktop view
This PR fixes #116

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by vinayaksh42** [comment]
Changes to the copyright text and social media icons are looking great.
Just one thing regarding the top (logo) section of the footer.
Before:
chrome_SI7tegHWnR
After:
chrome_ypl7S7xqB9

The alignment is a bit off in the after version. The `Resources` section is not coming right under the `Getting Started` section

File: docs/_sass/footer.scss [hunk 1]
```diff
@@ -1,304 +1,320 @@
...
+
+  }
+  .copyright-cont {
+    justify-content: center;
+    margin: 1rem 0rem;
+  }
+  .footer-columns {
+    display: flex;
+    justify-content: center;
+    padding-right: 50px;
```
Comment (JubayerJoy) [inline docs/_sass/footer.scss:291]: This should solve the alignment issue mentioned by @vinayaksh42 
```suggestion
    padding-right: 37px;
```

**Review by JubayerJoy — COMMENTED** [review]
Changes look good, just a small change :rocket:
Also, please resolve this [mixed content](https://github.com/layer5io/getnighthawk/pull/129/checks?check_run_id=2741670177) issue. Don't know why it's giving an error though, you didn't make any change or used it.

Anyways, Really nice work 🎈

**Comment by Jashpatel1** [comment]
@Anita-ihuman Please fix the conflicts.

**Comment by Jashpatel1** [comment]
@Anita-ihuman There are conflicting files present yet. Once you fix them, we are ready to go.

**Pull request merged**